src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+ul {
```

Comment:
<img width="1014" alt="image" src="https://github.com/user-attachments/assets/f5f6ac79-c2b7-471d-9f96-f18a2bba10ee">

Reply:
Ok. I got it. Thanks.